src/test/resources/Features/OCD-2916-BasicSurveillanceReporting.feature

```diff
@@ -0,0 +1,118 @@
+@Regression
+Feature: OCD-2916 Basic Surveillance Reporting
+  This test checks whether edit, split and merge developer functionality work as expected
+
+Scenario Outline: User logs in as ROLE and verifies navigation is successful to the Surveillance Report page
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the surveillance reports page
+    Then I see "CHPL Surveillance" as the page title for the surveillance report page
+    Examples:
+      |ROLE       |
+      |ROLE_ACB   |
+      |ROLE_ADMIN |
+      |ROLE_ONC   |
+
+Scenario Outline: User logs in as ROLE and verifies Surveillance reports are available for UL LLC, Drummond Group, ICSA Labs and SLI Compliance
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the surveillance reports page
+    Then I see available reports for "<ACBs>" 
+    Examples:
+      |ROLE         |ACBs                                             |
+      |ROLE_ADMIN   |Drummond Group ICSA Labs SLI Compliance UL LLC   |
+
+Scenario Outline: User logs in as ROLE and verifies years and quarters are listed for each ACB
+    Given I'm logged in as "<ROLE>"
+    When I navigate to the surveillance reports page
+    And I expand "<ACB>"
+    Then I see "<Year-Quarters>" listed for each ACB
+    Examples:
+      |ROLE       |ACB           |Year-Quarters           |
+      |ROLE_ADMIN |UL LLC        |Year Q1 Q2 Q3 Q4        |
+      |ROLE_ONC   |SLI Compliance|Year Q1 Q2 Q3 Q4        |
+      |ROLE_ADMIN |Drummond Group|Year Q1 Q2 Q3 Q4        |
+      |ROLE_ONC   |ICSA Labs     |Year Q1 Q2 Q3 Q4        |
+
```

Comment:
I think, instead of this and above scenario-checking list of ACBs & quarters, running following scenario to test complete workflow would be a good regression test - like expand it to cover post confirm action as well. The scenario outline does say 'initiate quarterly report' but stops at pop up verification